factory-hooks/post-settings-php/secrets.php

```diff
@@ -7,3 +7,9 @@
 if (file_exists($secrets_file)) {
   require $secrets_file;
 }
+
+$cloudNextSecretsFile = sprintf('%s/secrets.settings.php', $_ENV['HOME']);
+
+if (file_exists($cloudNextSecretsFile)) {
+  require $cloudNextSecretsFile;
+}
```

Comment:
Will there still be a file in the old location? If not, maybe the code to require the file in the old location should be removed.

Reply:
The file is indeed still there, but it seems the file isn't loading. I left the original code in place to see if this fixes the issue. If it works, I'll open a second PR to remove the old file path.